.github/workflows/enable-pack-build-and-test.yaml

```
@@ -0,0 +1,115 @@
+          echo "TOOLS_DIR=${{ github.workspace }}/exchange-tools" >> ${GITHUB_ENV}
+          echo "PACKS_PATH=${{ github.workspace }}/packs" >> ${GITHUB_ENV}
+
+      - name: Checkout ci repo
+        uses: actions/checkout@v2
+        with:
+          repository: StackStorm-Exchange/ci
+          ref: ${{ env.ci_branch }}
+          path: ci
+          fetch-depth: 1
+
+      - name: Checkout exchange-tools repo
+        uses: actions/checkout@v2
+        with:
+          repository: StackStorm-Exchange/exchange-tools
+          ref: ${{ env.exchange_tools_branch }}
+          path: exchange-tools
+          fetch-depth: 1
+
+      - name: Checkout Pack Repos and enable their Build and Test Workflow
+        shell: bash
+        env:
+          PACKS_ORG: ${{ env.packs_org }}
+          PACKS_PREFIX: ${{ env.pack_repo_prefix }}
+        run: |
+          mkdir -p ${PACKS_PATH}
+          cd ${PACKS_PATH}
+          source ${TOOLS_DIR}/functions.sh
+          for repo_name in $(_gh_list_repo_names ${PACKS_ORG} ${PACKS_PREFIX}); do
+            echo "::group::Clone ${PACKS_ORG}/${repo_name}"
+            gh repo clone "${PACKS_ORG}/${repo_name}"
+            if latestTag=$(git -C ${repo_name} describe --tags `git -C ${repo_name} rev-list --tags --max-count=1`); then
+              echo latestTag = $latestTag
+              git -C ${repo_name} checkout $latestTag -b latestTagBranch
+            fi
+            echo
+            echo "::endgroup::" # DELETED notices will not be folded to simplify scanning action output
+            if [[ -z "$latestTag" ]]; then
+              rm -rf ${repo_name}
+              echo "DELETED clone of ${PACKS_ORG}/${repo_name}: missing git tags"
+              echo "${PACKS_ORG}/${repo_name}" >> ${GITHUB_WORKSPACE}/bad_pack_repos
+            elif [[ ! -f "./${repo_name}/pack.yaml" ]]; then
+              rm -rf ${repo_name}
+              echo "DELETED clone of ${PACKS_ORG}/${repo_name}: missing pack.yaml"
+              echo "${PACKS_ORG}/${repo_name}" >> ${GITHUB_WORKSPACE}/bad_pack_repos
+            else
+              #
+              #  This is not a bad Pack, ensure the Build and Test workflow is active
+              #
+              ORG_REPO="${PACKS_ORG}/${repo_name}"
+              WORKFLOW="repos/${PACKS_ORG}/${repo_name}/actions/workflows/build_and_test.yaml"
+              WORKFLOW_NAME="CI/build_and_test"
+              echo "::group::Ensure ${WORKFLOW_NAME} workflow for ${PACKS_ORG}/${repo_name} is active"
+              if gh api --silent --method GET $WORKFLOW 2>/dev/null; then
+                echo "GitHub ${WORKFLOW_NAME} workflow found for ${PACKS_ORG}/${repo_name}"
+                if [[ "$(gh api --method GET $WORKFLOW --jq .state)" != "active" ]]; then
+                  echo "::notice::GitHub ${WORKFLOW_NAME} workflow for ${PACKS_ORG}/${repo_name} not active, enabling"
+                  gh workflow enable $WORKFLOW --repo ${PACKS_ORG}/${repo_name}
```

I don't think the API path will work here. Should this be something like this?

```suggestion
                  gh workflow enable ${WORKFLOW_NAME} --repo ${PACKS_ORG}/${repo_name}
```

---

> I don't think the API path will work here. Should this be something like this?

Ah yes, good catch.

---

It'll just be the top-level Workflow name I think:

```shell
❯ gh workflow list --repo stackstorm-exchange/stackstorm-jira
CI       active  16569399
Release  active  16569400
```